river-ridgelines/README.md

```diff
@@ -0,0 +1,11 @@
+# river-ridgelines
+
+To run this repo, change your working directory to `river-ridgelines` and run `tar_make()`:
```

Comment:
```suggestion
To run this repo, change your working directory to `river-ridgelines` by creating a new `.Rproj` file or with `setwd()`. Then run `targets::tar_make()`:
```

Reply:
Just a suggestion, but in my experience more folx would understand the option to create a new project rather than to change the working directory. I also think this is more in line with the way the other idea blitzes were set up?

Reply:
I may be wrong, but I don't _think_ you can create a new R project from a subdirectory of a Git repository and still use version control - @lindsayplatt is there a way to do so? I think this approach, of running `tar_make()` within a subdirectory of the repo is consistent with how the Chart Challenge repo, at least, was set up.

Reply:
The way that I set up my repo for idea blitz was to use version control in terminal, rather than RStudio, since in RStudio the project file was one directory level down from the git control. Each sub directory then had it's on project file, with one master git control at the idea-blitz/ directory level. But yes, otherwise agree that for "normal" repos it makes most sense to have the .rproj and git in the same directory.

Reply:
And yes, I did run `tar_make()` in my subdirectory. For the next idea blitz, @ceenell can you help clarify how to set these subdirectories and git and projects up so we all approach it the same way? I am pretty sure Anthony did it the same way I did, but maybe we did it wrong since we're the newbies?? 🤷

Reply:
Ah, I see - I was trying to think of how it would be done through R version control, but that approach of keeping git separate from the R projects seems like it would work, too. And does make it cleaner for running `targets` if you're primarily working in just one subdirectory.

Reply:
Sure thing, we can come up with a consistent way to approach this type of repo. The way I have done it is to create a new `.Rproj` in each subdirectory and create a new branch to work on.

Reply:
Cool, sounds like I was the odd one out then. I like that approach, so will switch to that.